tests/test_json.py

```diff
@@ -49,6 +50,23 @@ def test_encoding(input, output):
     assert output == json.dumps(input, default=pydantic_encoder)
 
 
+def test_path_encoding(tmpdir):
+    class PathModel(BaseModel):
+        path: Path
+        file_path: FilePath
+        dir_path: DirectoryPath
+
+    tmpdir = Path(tmpdir)
+    path = tmpdir / "foo"
+    file_path = tmpdir / "bar"
+    file_path.touch()
+    dir_path = tmpdir / "baz"
+    dir_path.mkdir()
+    model = PathModel(path=path, file_path=file_path, dir_path=dir_path)
+    expected = json.dumps({"path": str(path), "file_path": str(file_path), "dir_path": str(dir_path)})
```

Comment:
better to actually write this out rather than use `str` so it's explicit in the test what we're expecting.

Reply:
I think you might have missed this comment as i made it outside the review?

Reply:
Unsure of what you meant, considering my replies, but I went ahead and converted `expected` to a string, using `.format()` only when required.  It's as static as I can reliably make it while keeping portability.